### DISCUSSION TITLE
Prodigy is SCAMMING lmao

### DISCUSSION THREAD

**Discussion by afkvido** [Prodigy-Hacking/ProdigyMathGameHacking#1260] [(A) Anything]
## Prodigy is SCAMMING lmao
How to replicate:
Without the hacks installed, open the javascript console in prodigy,

ITS GOING TO TELL YOU YOU'RE BEING SCAMMED LMAO

You *could* get scammed through the console, but fr, you're just going to probably hack in some free gold ;p

**Comment by donaldli2020** [comment]
WARNING!
This is a browser feature for developers only.  If someone told you to copy-paste something here, it is a scam and may provide others with access to your account and/or your computer.  This may result in your account being deleted and/or you may be banned from Prodigy.

**Reply by YourOrdinaryProdigyHacker** [reply]
;o

**Reply by afkvido** [reply]
yup, that's what it says.
